audio_messages_en.txt

```diff
@@ -0,0 +1,90 @@
+mp3/0900_admin.mp3|Welcome to the Admin Menu. Please select a function with the volume keys and confirm it with the pause key! By a long press on the pause button you can cancel the process.
+mp3/0901_card_reset.mp3|Configure a new card
+mp3/0902_max_volume.mp3|Set maximum volume.
+mp3/0903_min_volume.mp3|Set minimum volume.
+mp3/0904_init_volume.mp3|Set volume at startup.
+mp3/0905_eq.mp3|Configure EQ.
+mp3/0906_modifiers.mp3|Create a modification card. These cards are placed after the actual music cards and change the behavior of your TonUINO.
+mp3/0907_shortcut.mp3|Configure keys with a shortcut. A shortcut is started by a long keystroke and only works if nothing is currently being played.
+mp3/0908_standbytimer.mp3|Configure the timer.
+mp3/0909_batch_cards.mp3|Create single cards for a folder.
+mp3/0910_switch_volume.mp3|Invert the function of the volume keys.
+mp3/0911_reset.mp3|Delete all settings.
+mp3/0912_admin_lock.mp3|Secure the admin menu.
+mp3/0913_pause_on_card_removed.mp3|Pause when card is removed.
+mp3/0919_continue_admin.mp3|and continuing in the admin menu
+mp3/0920_eq_intro.mp3|Please select a setting for the EQ with the volume keys and confirm it with the pause key.
+mp3/0921_normal.mp3|Normal
+mp3/0922_pop.mp3|Pop
+mp3/0923_rock.mp3|Rock
+mp3/0924_jazz.mp3|Jazz
+mp3/0925_classic.mp3|Classic
+mp3/0926_bass.mp3|Base
+mp3/0930_max_volume_intro.mp3|Select maximum volume and confirm with the pause key.
+mp3/0931_min_volume_into.mp3|Select minimum volume and confirm with the pause key.
+mp3/0932_init_volume_into.mp3|Select volume at startup and confirm with the pause key.
+mp3/0933_switch_volume_intro.mp3|Do you want to reverse the function of the volume buttons? You have to press the keys for a long time to go forward or backward one song.
+mp3/0934_no.mp3|No.
+mp3/0935_yes.mp3|Yes.
+mp3/0936_batch_cards_intro.mp3|OK, now please place the cards on the box one after the other. I will announce the respective number before, so that you don't get confused. To cancel, just press one of the volume buttons!
+mp3/0940_shortcut_into.mp3|Please select the shortcut you want to configure.
+mp3/0941_pause.mp3|Up and down key
+mp3/0942_up.mp3|Up key
+mp3/0943_down.mp3|Down key
+mp3/0944_startup.mp3|On startup
+mp3/0960_timer_intro.mp3|After how many minutes should the TonUINO be switched off?
+mp3/0961_timer_5.mp3|5 minutes.
+mp3/0962_timer_15.mp3|15 minutes.
+mp3/0963_timer_30.mp3|30 minutes.
+mp3/0964_timer_60.mp3|60 minutes.
+mp3/0965_timer_disabled.mp3|do not switch off automatically
+mp3/0970_modifier_Intro.mp3|Now please select your modification card with the volume keys.
+mp3/0971_modifier_SleepTimer.mp3|Snooze mode
+mp3/0972_modifier_FreezeDance.mp3|Stop Dance - TonUINO plays stop dance with you and randomly stops playback briefly for you.
+mp3/0973_modifier_Locked.mp3|Lock TonUINO
+mp3/0974_modifier_Toddler.mp3|Crawler mode - All keys of the TonUINO are locked for the very small ones. Cards still work.
```

Comment:
I think this should be called "Toddler mode" (like the filename suggests).

Reply:
Changed